benchmarks/python/conftest.py

```diff
@@ -96,6 +100,7 @@ def pytest_collection_modifyitems(session, config, items):
         compile = true: torch.compile benchmark
     """
     run_eager = config.getoption("--benchmark-eager")
+    run_thunder = config.getoption("--benchmark-thunder")
```

Comment:
This alone will not filter out the `thunder` benchmarks. The code following this removes the tests corresponding to that option.

By default, we are still running both.

Reply:
added and tested with `pytest ./benchmarks/python/test_groupnorm_fwd.py -s -v -k "bfloat16 and 512_2_4_4"`
Get:
```
[2024-08-09 08:08:45] benchmarks/python/test_groupnorm_fwd.py::test_groupnorm_fwd_nvf_benchmark[dtype=torch.bfloat16-size=[512_2_4_4]] PASSED
[2024-08-09 08:08:48] benchmarks/python/test_groupnorm_fwd.py::test_groupnorm_fwd_thunder_benchmark[dtype=torch.bfloat16-size=[512_2_4_4]] SKIPPED (need --benchmark-thunder option to run)
[2024-08-09 08:08:48] benchmarks/python/test_groupnorm_fwd.py::test_groupnorm_fwd_baseline_benchmark[dtype=torch.bfloat16-size=[512_2_4_4]-eager] SKIPPED (need --benchmark-eager option to run)
[2024-08-09 08:08:48] benchmarks/python/test_groupnorm_fwd.py::test_groupnorm_fwd_baseline_benchmark[dtype=torch.bfloat16-size=[512_2_4_4]-compile] SKIPPED (need --benchmark-torchcompile option to run)
```